[DO] Document GCP Secret Manager for Databricks PAR credentials (#38011)
DataDog/documentation: content/en/data_observability/jobs_monitoring/databricks/private_link.md

   1. Set the secret's lifetime in days (maximum 730 days).
   1. Click **Generate**.
   1. Make the Databricks credentials available to the runner using **one** of the following methods:

>       - **Cloud secret storage** (AWS Secrets Manager, AWS Parameter Store, Azure Key Vault, or GCP Secret Manager): Store the credentials as a JSON blob:

Reviewer: ```suggestion
      - **Cloud secret storage** (AWS Secrets Manager, AWS Parameter Store, Azure Key Vault, or Google Cloud Secret Manager): Store the credentials as a JSON blob:
```

        ```json
        {"client_id": "<CLIENT_ID>", "client_secret": "<CLIENT_SECRET>"}
        ```

>         Make note of the secret path (AWS ARN, AKV URL, or GCP resource name). You enter this in the integration tile or set it as the `DATABRICKS_SECRET_PATH` environment variable on the runner container or pod.

Reviewer: ```suggestion
        Make note of the secret path (AWS ARN, AKV URL, or Google Cloud resource name). You enter this in the integration tile or set it as the `DATABRICKS_SECRET_PATH` environment variable on the runner container or pod.
```

   1. Select the **Script** option, then click **Next**.
   1. Follow the instructions under **Docker**.

      **Important**: Replace `gcr.io/datadoghq/private-action-runner:v1.21.0-large` with `gcr.io/datadoghq/dd-data-observability-par:1.21.0-3`.

   1. The Private Action Runner should show up at the bottom as "successfully installed."
1. Configure credentials on the Private Action Runner using one of the following methods:
    - Set the `DATABRICKS_CLIENT_ID` and `DATABRICKS_CLIENT_SECRET` environment variables directly on the runner. Leave "Secret Path" in the integration tile blank.

>     - Use cloud secret storage. Ensure an identity is assigned to the instance ([Azure Managed Identity][5], [AWS IAM Role][6], or [GCE Service Account][7]) with permissions to read the secret created in [Step 2](#step-2-databrick-prerequisites), and provide the path to the secret either via the `DATABRICKS_SECRET_PATH` environment variable, or by providing it to the "Secret Path" field in the integration tile.

Reviewer: ```suggestion
    - Use cloud secret storage. Ensure an identity is assigned to the instance ([Azure Managed Identity][5], [AWS IAM Role][6], or [GCE Service Account][7]) with permissions to read the secret created in [Step 2](#step-2-databrick-prerequisites), and provide the path to the secret either through the `DATABRICKS_SECRET_PATH` environment variable, or by providing it to the "Secret Path" field in the integration tile.
```

>     - Use cloud secret storage. Ensure an identity is assigned to the instance ([Azure Managed Identity][5], [AWS IAM Role][6], or [GCE Service Account][7]) with permissions to read the secret created in [Step 2](#step-2-databrick-prerequisites), and provide the path to the secret either through the `DATABRICKS_SECRET_PATH` environment variable, or by providing it to the "Secret Path" field in the integration tile.

Reviewer: ```suggestion
    - Use cloud secret storage. Ensure an identity is assigned to the instance ([Azure Managed Identity][5], [AWS IAM Role][6], or [GCE Service Account][7]) with permissions to read the secret created in [Step 2](#step-2-databricks-prerequisites), and provide the path to the secret either through the `DATABRICKS_SECRET_PATH` environment variable, or by providing it to the "Secret Path" field in the integration tile.

```
